docs-v2/_docs/proxying.md

```diff
@@ -132,55 +132,78 @@ stubFor(get(urlEqualTo("/users/12345.json"))
   .withStatus(503)));
 ```
 
-wiremock-jre8 allows forward proxying, stubbing & recording of HTTPS traffic. By default, this will verify the origin certificates; this behaviour can be turned to trusting all as follows:
+### Browser proxying of HTTPS
+
+wiremock-jre8 allows forward proxying, stubbing & recording of HTTPS traffic.
+
+This happens automatically when browser proxying is enabled.
+
+*We strongly recommend using WireMock over HTTP to proxy HTTPS*; there are no associated security concerns, and proxying HTTPS over HTTPS is poorly supported by many clients.
+
+Note that when clients / operating systems distinguish between HTTP & HTTPS proxies they are often referring to the scheme of the target server, not the scheme the proxy server is listening on.
+
+#### Getting your client to trust the certificate presented by WireMock
+
+Normally when proxying HTTPS the proxy creates a TCP tunnel between the client and the target server, so the HTTPS session is between the client and the target server.
+While the proxy passes the bytes back and forward, it cannot understand them because there is end-to-end encryption between the client and the target.
+
+WireMock needs to decrypt the traffic in order to record or replace it with stubs.
+Consequently, there have to be two separate HTTPS sessions - one between WireMock and the target server, and one between the client and WireMock.
+This means that when you request https://www.example.com proxied via WireMock the HTTPS certificate will be presented by WireMock, not www.example.com.
+Inevitably it cannot be trusted by default - otherwise no internet traffic would be secure.
+
+WireMock uses a root Certificate Authority private key to sign a certificate for each host that it proxies.
+By default, WireMock will use a CA key store at `$HOME/.wiremock/ca-keystore.jks`.
+If this key store does not exist, WireMock will generate it with a new secure private key which should be entirely private to the system on which WireMock is running.
+You can provide a key store containing such a private key & certificate yourself using the `--ca-keystore`, `--ca-keystore-password` & `--ca-keystore-type` options.
+> See [this script](https://github.com/tomakehurst/wiremock/blob/master/scripts/create-ca-cert.sh)
+> for an example of how to build a key & valid self-signed root certificate called
+> ca-cert.crt already imported into a keystore called ca-cert.jks.
+
+This CA certificate can be downloaded from WireMock's `/__admin/recorder` endpoint.
```

Comment:
Please could you also provide the URL for directly downloading it, and indicate that /recorder is a web page?

Reply:
Done